framerail/src/lib/server/load/page.ts

```diff
@@ -21,11 +21,17 @@ export async function loadPage(
   const sessionToken = cookies.get("wikijump_token")
   let locales = parseAcceptLangHeader(request)
 
-  // Request data from backend
-  const response = await pageView(domain, locales, route, sessionToken)
-
   // TODO insert user preference at the beginning of the list
 
+  // Request data from backend
+  // Includes fallback locale in case there is no Accept-Language header
+  const response = await pageView(
+    domain,
+    [...locales, defaults.fallbackLocale],
```

Comment:
This works, but I can't help but feel like it's strange to always have a fallback to English there even if it was never requested by the user. Would `if empty then [default_locale]` make sense to you?

Reply:
This is done in a similar fashion to fallbackLocale always being present in the later translation request when the entire locale list is constructed by (user preference ->) browser header -> site settings -> fallback locale. Here we only have the browser header and fallback locale.